utils/python/CIME/XML/GenericXML.py

```diff
@@ -84,11 +84,16 @@ def get_value(self, item):
         get_value is expected to be defined by the derived classes, if you get here it is an error.
         """
         logging.debug("Get Value for "+item)
+        result = None
         if item in self.lookups.keys():
-            return self.lookups[item] 
+            result = self.lookups[item]
         if item in os.environ:
-            return os.environ.get(item)
-#        expect(False, "Not implemented")
+            result = os.environ.get(item)
+
```

Comment:
I think that this warning may be a problem unless we add a has_value function to test whether the variable exists.  The reason is that at the Case level I am always searching all of the objects until I find the variable defined, so None is the expected result pretty frequently.